Autopilot/Inc/gps.hpp

```diff
@@ -55,7 +55,7 @@ class Gps
 		* or with old data (in case of old data, the dataIsNew flag of the result struct will be cleared).
 		* @param[out]		Data 		pointer to the result struct.
 		*/
-        virtual void GetResult(GpsData_t *Data) = 0;
+        virtual void GetResult(GpsData_t &Data) = 0;
```

Comment:
pls fix to match style guide: https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/1827635203/Style+Guide